Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -471,7 +461,7 @@ theorem mul_inv_cancel_of_le_one [h : IsDedekindDomain A] {I : Ideal A} (hI0 : I
   · rw [← hJ, hJ1, coeIdeal_top]
   obtain ⟨x, hx, hx1⟩ :
     ∃ x : K, x ∈ (J : FractionalIdeal A⁰ K)⁻¹ ∧ x ∉ (1 : FractionalIdeal A⁰ K) :=
-    exists_not_mem_one_of_ne_bot hNF hJ0 hJ1
+    exists_not_mem_one_of_ne_bot hJ0 hJ1
```

Comment:
I don't know whether we can golf this proof using the `not_inv_le_one_of_ne_bot` version.